examples/resources/datadog_integration_aws_account/resource.tf

```diff
@@ -0,0 +1,44 @@
+# Create new integration_aws_account resource
+
+resource "datadog_integration_aws_account" "foo" {
+  account_tags   = ["env:prod"]
+  aws_account_id = "123456789012"
+  aws_partition  = "aws"
+  aws_regions {
+    include_all = true
+  }
+
+  auth_config {
+    aws_auth_config_role {
+      role_name = "DatadogIntegrationRole"
+    }
+  }
+  logs_config {
+    lambda_forwarder {
+      lambdas = ["arn:aws:lambda:us-east-1:123456789012:function:my-lambda"]
+      sources = ["s3"]
+    }
+  }
+  metrics_config {
+    automute_enabled          = true
+    collect_cloudwatch_alarms = true
+    collect_custom_metrics    = true
+    enabled                   = true
+    namespace_filters {
+      exclude_only = ["AWS/SQS", "AWS/ElasticMapReduce"]
+    }
+    tag_filters {
```

Comment:
Is this right? I'm not familiar with the terraform format. This looks like tag_filters is a single thing with "namespace" and "tags" fields, but it's really a list, right? Is this how a singleton of an object is represented?

Reply:
Discussed offline, this is correct.